index.html

```diff
@@ -103,7 +137,7 @@ <h3 class="recentWorkTitle">Multi-Post Stories</h3>
           <li class="tech">Bootstrap</li>
           <li class="tech">Ruby</li>
         </ul>
-        <button class="mybtn" type="button">See Project</button>
+        <button class="mybtn " id="recentbutton" type="button">See Project</button>
```

Comment:
- [ ] [OPTIONAL] When I hover the button in the popup window, your content moves, kindly fix it 😉 

![image](https://user-images.githubusercontent.com/60164239/132774140-ffcf841f-35e1-4295-b4d8-12d3678ebfa7.png)

Reply:
I will do it. thanks